src/component/component/index.js

```diff
@@ -137,19 +137,31 @@ export class Component extends BaseComponent {
         // Register all of the drivers for instantiating components. The model used is -- there's a standard javascript
         // way of rendering a component, then each other technology (e.g. react) needs to hook into that interface.
         // This makes us a little more pluggable and loosely coupled.
-
+        this.driverResults = {};
```

Comment:
I meant that:
```
imports: [MyLoginXComponent
  .driverResults
  .angular2
  .module]
```

Is accessing the module the same way as:
```
imports: [MyLoginXComponent
   .driver('angular2', { Component, NgModule, ElementRef, NgZone, BrowserModule})
  .driverResults
  .angular2
  .module]
```

and that:
```
MyLoginXComponent
   .driver('angular2', { Component, NgModule, ElementRef, NgZone, BrowserModule});

imports: [MyLoginXComponent
  .driverResults
  .angular2
  .module]
```
The only difference is not initializing the driver because it has been done already. 

Looks like I haven't seen the implementation of all drivers and looks like they are different because I haven't seen this in angular driver:
 ```
let frameworkComponent = MyComponent.frameworkName;
```